src/scenic/syntax/translator.py

```diff
@@ -985,19 +1029,32 @@ def wrapStatementCall():
 					inConstructorContext = True
 					allowedPrefixOps = self.specifiersForConstructor(context)
 				else:
-					for opTokens, op in infixTokens.items():
-						if not op.contexts or context in op.contexts:
-							allowedInfixOps[opTokens] = op.token
 					for name, mod in modifierNames.items():
 						if not mod.contexts or context in mod.contexts:
 							allowedModifiers[name] = mod.name
 					if context in modifierNames:
 						allowedTerminators = modifierNames[context].terminators
 					elif context in terminatorsForStatements:
 						allowedTerminators = terminatorsForStatements[context]
+			# construct allowedInfixOps
+			if context:
+				# all function calls on the stack
+				contexts = set(ctx for ctx, _ in functionStack)
+				for opTokens, op in infixTokens.items():
+					# if not context is specified, the operator can be used
+					if not op.contexts:
+						allowedInfixOps[opTokens] = op.tokens
+						continue
+					# if operator is start level only and it is start level, cannot be used
```

Comment:
"it is" -> "it is not"

Reply:
Fixed in 8da575ad72765f9fb3602857d56148ce4070e755